lib/message.js

```diff
@@ -0,0 +1,81 @@
+const uuid = require('uuid');
+const zlib = require('zlib');
+
+class Message {
+  constructor(msg) {
+    this.requestId = msg.requestId || uuid.v4();
+    this.messageId = msg.messageId || uuid.v4();
+    this.from = msg.from || 'unknown';
+    this.to = msg.to || 'unknown';
+    this.timestamp = new Date();
+    this.priority = msg.priority || 1;
+    this.body = msg.body || '';
+    this.correlationId = msg.correlationId;
+    this.replyTo = msg.replyTo;
+    this.ttl = msg.ttl || 0;
+    this.type = msg.type || 'msg';
+  }
+
+  set setMessageId(value) {
+    this.messageId = value;
+  }
+
+  get getMessageId() {
```

Comment:
setter and getter look like props, and are meant to be used like
```js
Message.setMessageId = 'something' // setter
const msgId = Message.getMessageId // getter
```
So could you please get some better name for both to not look like functions